components/checkbox/stories/template.js

```diff
@@ -50,7 +50,7 @@ export const Template = ({
 					typeof size !== "undefined",
 				[`${rootClass}--emphasized`]: isEmphasized,
 				["is-indeterminate"]: isIndeterminate,
-				["is-disabled"]: isDisabled|| isReadOnly,
+				["is-disabled"]: isDisabled,
```

Comment:
I like that we were able to remove the `isReadOnly` arg from this. It immediately makes sense now to any newcomer to the codebase.